semilla_moneda/semilla_moneda.py

```diff
@@ -8,28 +8,29 @@
 
 def main():
     ### INPUT DE ENTROPÍA BINARIA
-    binario = input("Introduce los 128, 160, 192, 224 o 256 bits de entropia: ")
+    binario = input("Introduce los 128, 160, 192, 224 o 256 bits de entropía: ")
     # Ejemplo 256 bits
     # binario = '0011001010000101011111010000101111111111101000001001000001001010110100010101111001001011000100111100011110001001111011110111011010010100110011001110111001100010111011010010101101010011110100100110101111110001100101011001000110100010000110110001100101110001'
     ## Ejemplo 128 bits
     # binario = '00110010100001010111110100001011111111111010000010010000010010101101000101011110010010110001001111000111100010011110111101110110'
 
     ### DEFINIENDO VARIABLES DEL BIP39
     binario_len = len(binario)  # ENT en el BIP39
+    print(
+        f"\nLa longitud de la entropía introducida es de: {binario_len}"
+    )
+    if binario_len != 128 and binario_len != 160 and binario_len != 192 and binario_len != 224 and binario_len != 256:
+        raise Exception("Entropía inválida. Verifica que tu entropía tenga una de estas longitudes: 128, 160, 192, 224, 256")
+
```

Comment:
I'd define the expected values as a list, and just check pertinence:

```suggestion
entropias = [128, 160, 192, 224, 256]
if binario_len not in entropias:
 raise ...
``` 

You can even use that list to print the values in the exception (just print the list)

Reply:
Done.